test/server.test.js

```diff
@@ -32,6 +33,9 @@ injectGlobal`
     display: flex;
     justify-content: center;
   }
+  .should-be-included {
+    content: 'css-foo matches regex and this rule is swallowed (current bug)'
+  }
 `
```

Comment:
No worries, just a minor edge case. I guess you could leave the test in pending (`xtest()`?) and fix it if/when you want to support